src/crimson/osd/osd_operation.h

```diff
@@ -146,6 +135,18 @@ class Operation : public boost::intrusive_ref_counter<
     }
   }
 
+protected:
```

Comment:
why reorder these declarations?

Reply:
It's to hide `add_blocker()` and `clear_block()` into private, since `with_blocking_future` is much better and exception-safe.

Reply:
i see, could you move the private members to the end of the class. see https://google.github.io/styleguide/cppguide.html#Declaration_Order , add note the rationale in the commit message? as understanding "why" is probably also important as "what" when reviewing the change.

Reply:
Added commit message, and also split the `Operation` related changes.